src/main/java/listeners/MainEventListener.java

```diff
@@ -0,0 +1,232 @@
+     * @param member The Member to reply to
+     * @param channel The Channel to reply within
+     * @param server The Server to check in
+     * @param topic The Topic to check for
+     * @return Contains the Topic, if it exists
+     */
+    private static Optional<Topic> checkTopicExists(Member member, TextChannel channel, Server server, String topic) {
+        Optional<Topic> topicOptional = Arrays.stream(server.getTopics())
+                .filter(t -> t.getName().equalsIgnoreCase(topic))
+                .findFirst();
+        if (topicOptional.isEmpty()) {
+            channel.sendMessage(String.format(
+                    "%s Topic \"%s\" does not exist.",
+                    member.getAsMention(),
+                    topic)).queue();
+        }
+        return topicOptional;
+    }
+
+    @Override
+    public void onReady(@NotNull ReadyEvent event) {
+        System.out.println("Logged in");
+        event.getJDA().getPresence().setPresence(Activity.playing("$help"), false);
+    }
+
+    @Override
+    public void onGuildMessageReceived(@NotNull GuildMessageReceivedEvent event) {
+        String[] tokens = event.getMessage().getContentDisplay().split(" ");
+        if (!tokens[0].startsWith("$")) return;
+
+        String guildID = event.getGuild().getId();
+        Server server = servers.computeIfAbsent(guildID, k -> Server.load(event.getGuild()));
+
+        // pick the correct method to call
+        CommandHandler commandHandler;
+        switch (tokens[0].substring(1)) {
+            case "help" -> commandHandler = this::help;
+            case "maketopic" -> commandHandler = this::maketopic;
+            case "deletetopic" -> commandHandler = this::deletetopic;
+            case "showtopics" -> commandHandler = this::showtopics;
+            case "queue" -> commandHandler = this::queue;
+            case "ready" -> commandHandler = this::ready;
+            case "showqueue" -> commandHandler = this::showqueue;
+            case "clear" -> commandHandler = this::clear;
+            default -> commandHandler = this::unknownCommand;
+        }
+        commandHandler.handle(
+                Objects.requireNonNull(event.getMember()),
+                event.getChannel(),
+                server,
+                Arrays.copyOfRange(tokens, 1, tokens.length));
+    }
+
+    private void help(Member member, TextChannel channel, Server server, String[] args) {
+        EmbedBuilder embedBuilder = new EmbedBuilder();
+        embedBuilder.setTitle("Help!");
+        embedBuilder.setDescription("Possible commands:");
+        embedBuilder.setColor(0xE57D25);
+
+        embedBuilder.addField("$queue <topic>", "Add yourself to a queue.", false);
+        embedBuilder.addField("$showqueue <topic>", "Show the people currently in queue.", false);
+        embedBuilder.addField("$showtopics", "List all topics.", false);
+        embedBuilder.addField("$ready <topic> (admin only)", "Retrieve the next person from the queue.", false);
```

Comment:
Is there a way to only show these commands to users with the admin role?

Reply:
(Out of scope but just to get some discussion)

Reply:
There should be, but it has a lot of overlap with #5. I will bundle these suggestions into that issue.

Reply:
Actually, on second thought, this would need to show three versions of the help list: one for normal users, one for mentors, and one for admins. In that case, it wouldn't make sense to merge it with #5. I'll make it into a separate issue.

Reply:
Included in #8.